styles/partials/variables.scss

```
@@ -1,6 +1,12 @@
+// Colors
 $primary-color: #111;
 
```

As mentioned before consider using a more descriptive variable name
```suggestion
$primary-color: #111;
$bg-color-light: #fff;
```

---

I've added the variable